movie_REST_API/movie_REST_API/App/Controllers/MovieViewController.swift

```diff
@@ -198,57 +169,64 @@ final class MovieViewController: UIViewController {
             rateButton.heightAnchor.constraint(equalToConstant: 40),
         ])
     }
-    
+
     private func setConstraintImage() {
         NSLayoutConstraint.activate([
             baseImageView.topAnchor.constraint(equalTo: rateButton.bottomAnchor, constant: 10),
             baseImageView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 10),
             baseImageView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -10),
             baseImageView.heightAnchor.constraint(equalToConstant: 200)
-            ])
+        ])
     }
-    
+
     private func setConstraintNewButton() {
         NSLayoutConstraint.activate([
             newButton.topAnchor.constraint(equalTo: view.topAnchor, constant: 300),
             newButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 15),
             newButton.heightAnchor.constraint(equalToConstant: 35),
             newButton.widthAnchor.constraint(equalToConstant: 120)
-            ])
+        ])
     }
 }
 
-// MARK: - Подписываемся на делегаты UITableViewDelegate, UITableViewDataSource
-extension MovieViewController: UITableViewDelegate, UITableViewDataSource {
+// MARK: - UITableViewDataSource
+
+extension MovieViewController: UITableViewDataSource {
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        movieViewModel.numberOfRowsInSection(section: section)
+        presenter?.numberOfRowsInSection(section: section) ?? 0
```

Comment:
```suggestion
        presenter?.numberOfRowsIn(section: section) ?? 0
```
section дублировать название не хорошо